src/utils/styled.ts

```diff
@@ -1,5 +1,10 @@
 import isPropValid from "@emotion/is-prop-valid";
-import { StyledOptions } from "@emotion/styled";
+
+export interface StyledOptions<Props = Record<string, any>> {
```

Comment:
This generic is unused

Reply:
Ye, but it is that way because of `StyledOptions` that we were importing before, which technically in there lib is not being utilized seems at all. Although this shouldn't be a final solution, as the issue still persist